### ISSUE TITLE
Moved from Windows to macOS, holding modifier key doesn't keep the action bar open

### ISSUE THREAD

**Issue by EthanHamric** [tullamods/Dominos#574]
**Addon Version**
9.1.0-Retail

**World of Warcraft Version**
Retail

**macOS Big Sur**
11.4

On Action Bar 1, under paging, I have shift bound to show action bar 2, and ctrl bound to show action bar 3. When I was still running windows, I could hold down shift and it would stick to action bar 2, meaning I could then use shift-1 to shift-+ on action bar 2, as long as I held down shift.

On macOS, I hold down shift, the action bar switches to 2, and I press, say, ability 3. A soon as I do that, several things happen: if I have all my addons enabled, shift-3 spell on bar 2 happens, then dominos briefly goes back to action bar 1, then I can press another ability using a shift chord, but then, even while holding shift, only action bar 1 abilities will work.

I then disabled everything but Dominos and Dominos Config and reloaded the UI. When I hold shift, I see action bar 2, as intended. When I keep holding shift and press any of 1 through +, the correct ability fires, but then the action bar switches to 1, and I have to press shift again to bring back action bar 2. Sometimes, though, it toggles between action bar 1 and 2 rapidly, but this only seems to happen for a few key presses.

I did see the sticky bar option in settings, but it doesn't really seem to make a difference if it's on or off, even when I reload the UI.

**Comment by Tuller** [comment]
What happens with the stock action bars? I think this one is Mac specific issue

**Comment by EthanHamric** [comment]
You're right, it's some sort of Mac specific issue. I just tried making a bunch of macros to do this, and I still have the same issue. It's like it ignores my modifier key after a couple of presses. Maybe I can find something in keyboard settings that will fix it :/

Thanks for taking a look at this!

I'll update by the end of the week, even if I don't find a workaround

**Comment by EthanHamric** [comment]
Well, it seems to be working again after running: defaults write com.blizzard.worldofwarcraft disable-expose-fix YES

It looks to be an issue with the way macOS handles the razer naga buttons. When I was in keybinding mode, holding shift and pressing 1 on my naga would bind 1 only. When I used the non numpad keyboard keys, shfit+1 bound as expected.

After I ran that terminal command, pressing shift and 1 on my naga now binds shift+1 as expected. I have no idea what this actually does, only that it seems to have fixed it. Here's the forum post I found this in: https://us.forums.blizzard.com/en/wow/t/razer-naga-and-alt-keymod-macros/155314/4
